components/Carbon.js

```diff
@@ -22,6 +22,20 @@ class Carbon extends PureComponent {
     this.handleLanguageChange = this.handleLanguageChange.bind(this)
     this.handleTitleBarChange = this.handleTitleBarChange.bind(this)
     this.codeUpdated = this.codeUpdated.bind(this)
+    this.codeMirrorUpdate = this.codeMirrorUpdate.bind(this)
+  }
+
+  codeMirrorUpdate(){
+    if(this.props.config.lineNumberSeperator){
```

Comment:
I know i mentioned calculating the color, but this seems like a lot of overhead for a small feature. Is there a color that works well with all the themes? Like `#fff`?

Sorry for making you do this